seatunnel-connectors-v2/connector-milvus/src/main/java/org/apache/seatunnel/connectors/seatunnel/milvus/catalog/MilvusCatalog.java

```diff
@@ -348,6 +348,9 @@ private FieldType convertToFieldType(Column column, PrimaryKey primaryKey) {
     @Override
     public void dropTable(TablePath tablePath, boolean ignoreIfNotExists)
             throws TableNotExistException, CatalogException {
+        if (!tableExists(tablePath) && !ignoreIfNotExists) {
+            throw new TableNotExistException(catalogName, tablePath);
+        }
         this.client.dropCollection(
                 DropCollectionParam.newBuilder()
```

Comment:
if `ignoreIfNotExists == true`, why we should continue execute `dropCollection`?